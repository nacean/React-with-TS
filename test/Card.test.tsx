// import React from "react";
// import { render } from "@testing-library/react";
// import Card from "../src/Card";

// describe("Card Template test", () => {
//   const component = render(
//     <Card number={123123135} cvc={12} date={"08/24"}></Card>
//   );

//   expect(component.container).toMatchSnapshot();

//   it("number", () => {
//     expect(component.getByText(/number : (\d{5})/g).innerHTML).toMatch(/\d{5}/);
//     expect(component.getByText(/cvc : (\d{2})/g).innerHTML).toMatch(/\d{2}/);
//   });
// });
