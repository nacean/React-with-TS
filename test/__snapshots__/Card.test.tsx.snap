// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <div>
    number : 
    123123135
    cvc : 
    12
    date : 
    08/24
  </div>
</div>
`;
